src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -693,10 +693,11 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
                     cmd, vnet_name, location, tags, vnet_address_prefix, subnet, subnet_address_prefix))
 
         if nsg_type == 'new':
-            nsg_rule_type = 'rdp' if os_type.lower() == 'windows' else 'ssh'
+            if nsg_rule is None:
```

Comment:
is it a bug fix? looks like before this PR, user cannot choose nsg_rule_type when they are creating a new nsg.

Reply:
Yes, it is. Now users can choose 22 or 3389.